### PR TITLE
[DOC-11930] Moved Eventing nav to a partial in the docs-devex repo

### PR DESCRIPTION
**Pull request by julia-cb**
Moved Eventing nav to a partial in the docs-devex repo.

docs-devex PR: https://github.com/couchbaselabs/docs-devex/pull/151
preview site: https://preview.docs-test.couchbase.com/eventingnav/server/current/eventing/eventing-overview.html

### CODE REVIEW

**Comment by julia-cb** [comment]
@osfameron @simon-dew Hey guys, I moved the Eventing nav to docs-devex [here](https://github.com/couchbaselabs/docs-devex/pull/151) and added the new partial file path to the nav.adoc file in this PR. Can you take a look and let me know if I did it properly? I don't want to merge it and break staging by accident lol. Thanks!

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -28,7 +28,9 @@ include::search:partial$nav.adoc[]
   ** xref:sdk:sdk-doctor.adoc[Troubleshooting Connections with SDK Doctor]
 
 include::n1ql:partial$nav.adoc[]
+
```
Comment (simon-dew) [inline modules/ROOT/nav.adoc:31]: Not sure about adding blank lines in the navigation list, they're fickle things, and the blank lines may have unwanted effects on the indenting of included partials. Have you done a preview build to check that it all works?

**Review by simon-dew — APPROVED** [review]
I'm going to approve this but please do a preview build to check that it works.

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (julia-cb) [reply modules/ROOT/nav.adoc:31]: I checked the navigation for Search and Query and everything works! I'll go ahead and merge this PR.

Preview here: https://preview.docs-test.couchbase.com/eventingnav/server/current/eventing/eventing-overview.html

**Pull request merged**